packages/web/src/shared/components/form/AutocompleteInput.tsx

```diff
@@ -80,23 +81,25 @@ const AutocompleteInput = forwardRef<HTMLInputElement, AutocompleteInputProps>(
 
     const root = useRef<HTMLDivElement>(null);
     useEffect(() => {
-      const handler = (e: PointerEvent) => {
-        if (!root.current?.contains(e.target as Element)) {
-          let newValue;
-          if (typeof activeIndex === 'number') {
-            newValue = filteredSuggestions[activeIndex];
-          } else {
-            newValue = input;
-          }
-          if (newValue !== value) {
-            onChange(newValue, true);
+      if (isFocused) {
+        const handler = (e: PointerEvent) => {
+          if (!root.current?.contains(e.target as Element)) {
+            let newValue;
+            if (typeof activeIndex === 'number') {
```

Comment:
I think our code would be much more readable if we defined a common helper function `isDefined`, and used it throughout our codebase.

```
export const isDefined = (value) => typeof value !== 'undefined';
```

Reply:
If it was a more complex expression other than one comparison, I'd agree, but in this case, I don't think the cost of introducing a utility that you have to import is worthwhile.